packages/zql/src/query/test/query-delegate.ts

```diff
@@ -39,6 +39,14 @@ export class QueryDelegateImpl implements QueryDelegate {
     this.#sources = sources ?? makeSources();
   }
 
+  get lc() {
+    return lc;
+  }
+
+  get slowMaterializationThreshold() {
+    return undefined;
+  }
+
```

Comment:
Maybe refactor this to:


```ts
onSlowMaterialization(duration: number, ast: ...): void;
```

Then the LogContext does not have to be part of the interface and the actual logging is handled by the delegate.

Reply:
Good idea, will do this!